### ISSUE TITLE
Problem of finetune on my data with detector COCO ckpt

### ISSUE THREAD

**Issue by JiahaoXia** [google/automl#959]
Follow the tutorial 8 Finetune on PASCAL VOC 2012 with detector COCO ckpt, I want to train efficientdet-d0 on my data. 
When I set var_freeze_expr: '(efficientnet|fpn_cells|resample_p6)' in the configure file, the final result is really bad (best object AP=0.019).
Then I remove the var_freeze_expr, the performance of the final model is improved a lot (best object AP>0.6).
Could anyone explain this, the tutorial 8 can get a good result by setting var_freeze_expr?

**Comment by ameyparanjape** [comment]
What `var_freeze_expr` did you set?
If you set `(efficientnet)`, that means the efficientnet `backbone` weights trained on COCO data are frozen and the model `head` is `trainable`. So the model `head` is learning from the PASCAL VOC 2012 data and adapting the weights to minimize the loss. Even though efficientnet `backbone` is trained on COCO data, it has learnt the fundamental features well, so it is making use of those features and hence the result is good after finetuning `head`. Please refer to the paper if you are interested in details of the architecture

**Comment by landskris** [comment]
Hi @ameyparanjape. 
Out of curiosity, would you know what precisely the `'resample_p6` does? 
If using (`'efficientnet'`) or (`'efficientnet'`|`'fpn_cells'`) I interpret that either the backbone weights or the backbone + fpn weights are frozen.  However, it is not clear to me what the resample layer refers to? Is it the level of which the freezing applies in the backbone?  E.g `resample_p5` would not freeze backbone or fpn weights in the P6 and P7 layers?
